src/captureStartupPerformance.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as vscode from 'vscode';
+import { delay } from 'lodash';
+
+export class CaptureStartupPerformance {
+  constructor() {
+    console.log('Constructor - This is the Capture Startup Performance class!');
+  }
+}
+
+export const captureStartupPerformance = async (): Promise<void> => {
+  console.log('Method - This is the Capture Startup Performance class!');
+
+  try {
+    // Show running notification to the user
+    await vscode.window.withProgress({
+      location: vscode.ProgressLocation.Notification,
+      title: 'SFDX: Capture Startup Performance',
+      cancellable: false
+      }, async (progress) => {
+      progress.report({ message: 'Running...' });
+
+      // open Developer: Startup Performance (open via API - try "perfview.show")
+      await openStartupPerformanceFile();
+      // save startup performance file (TODO: how to do this without popup dialog?)
+      const fileName = await saveCurrentFile();
+      // read startup performance file using fs.readFileSync()
+      const fileContents = await readStartupPerformanceFile(fileName);
+      // parse startup performance file using marked
+      await sleep(2000);
+      const parsedResults = await parseStartupPerformanceFile(fileContents);
+      // save parsed results to a file
+      await copyContentsToFile(parsedResults);
+      const parsedResultsFile = await saveCurrentFile();
+      // send parsed results to appinsights
+      await sendToAppInsights(parsedResultsFile);
+
+      await sleep(5000);
+    });
+
+    // Show success notification to the user
+    vscode.window.showInformationMessage('SFDX: Capture Startup Performance command completed successfully.');
```

Comment:
Why a separate notification for completion?

Reply:
I'm making this consistent with all our other Salesforce extensions.  When our other SFDX commands complete successfully, we get a notification indicating that.

(Although, I am quite surprised that the notification is now replaced with a message in the status bar now.)
![Screenshot 2024-08-21 at 10 47 58 AM](https://github.com/user-attachments/assets/16331657-e0eb-4c90-8e67-ed8bea377063)